etc/profile-a-l/freetube.profile

```diff
@@ -0,0 +1,32 @@
+# Firejail profile for freetube
+# Description: Youtube client with local subscription feature
+# This file is overwritten after every install/update
+# Persistent local customizations
+include freetube.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/FreeTube
+
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+# include disable-shell.inc -Creates problems with --appimage
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/FreeTube
+whitelist ${HOME}/.config/FreeTube
+
+shell none
+ignore seccomp
+seccomp !chroot
```

Comment:
:question: :question:

Reply:
electron profile has `seccomp` in it?

Reply:
```
ignore seccomp
seccomp !chroot
```
=> no seccomp at all

```
seccomp !chroot
seccomp
```
=> seccomp with default-filter without chroot.

Reply:
Yeah but shouldn't it block all since electron profile has it?

Reply:
https://github.com/netblue30/firejail/commit/3d8f587cd8e2604df928be21c4dd201bd0b818fc